pygfx/objects/_base.py

```diff
@@ -294,63 +257,65 @@ def receive_shadow(self, value):
         self._store.receive_shadow = bool(value)
 
     @property
-    def parent(self):
-        """Object's parent in the scene graph (read-only).
-        An object can have at most one parent.
-        """
-        return self._parent_ref and self._parent_ref()
+    def material(self):
+        """Wheter is object is rendered or not. Default True."""
+        return self._store.material
+
+    @material.setter
+    def material(self, material):
+        self._store.material = material
 
     @property
-    def children(self):
-        """The child objects of this wold object (read-only tuple).
-        Use ``.add()`` and ``.remove()`` to change this list.
-        """
-        return tuple(self._children)
+    def parent(self) -> "WorldObject":
+        if self._parent is None:
```

Comment:
I think you've lost the docstrings along the way somewhere

Reply:
Yep. I will reintroduce them once `pygfx.linalg` is no longer in use and the code itself settles (avoids spending time documenting functions that might still change before the PR is done).

Reply:
Yep. Will reintroduce once the code itself is starting to settle.